sdk/core/azure-core/inc/azure/core/internal/tracing/tracing_impl.hpp

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Handling log messages from Azure SDK.
+ */
+
+#pragma once
+
+#include "azure/core/datetime.hpp"
+#include "azure/core/internal/extendable_enumeration.hpp"
+#include "azure/core/nullable.hpp"
+#include "azure/core/tracing/tracing.hpp"
+#include "azure/core/url.hpp"
+#include <array>
+#include <memory>
+#include <string>
+#include <vector>
```

Comment:
Are all these include headers actually needed? It seems like at least `<vector>` isn't.

Reply:
url, array, and vector didn't need to be there, map did.